daemon/tests/happy_path.rs

```diff
@@ -1,9 +1,13 @@
-use crate::harness::flow::{is_next_none, next_cfd, next_order, next_some};
+use std::time::Duration;
+
+use crate::harness::flow::{is_next_none, next, next_cfd, next_order, next_some};
 use crate::harness::{assert_is_same_order, dummy_new_order, init_tracing, start_both};
+use daemon::connection::ConnectionStatus;
 use daemon::model::cfd::CfdState;
 use daemon::model::Usd;
 use maia::secp256k1_zkp::schnorrsig;
 use rust_decimal_macros::dec;
+use tokio::time::sleep;
```

Comment:
I'd prefer to keep all `use` statements in a single block.

Reply:
I'll try to keep an eye on it (or maybe write a lint against it), this is unintentional outcome of running a code action in my emacs ;/
fixed!